.circleci/config.yml

```diff
@@ -78,18 +99,38 @@ workflows:
           filters:
             tags:
               only: /.*/
+      # - test:
+      #     requires:
+      #       - install
+      #     filters:
+      #       tags:
+      #         only: /.*/
```

Comment:
Is there a reason these are commented out?

Reply:
@sofiapoh yeah the current snapshots are failing - if you can check / fix the snapshots and push them to this branch, I can add it to CI 👍

Reply:
alright!